+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Fri Nov 11 21:24:17 2022 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL 'freeradius freeradius-utils'
+ pkgs='freeradius freeradius-utils'
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs 'freeradius freeradius-utils' --node 1 --tempfile ''
+ tmpfile2='Last metadata expiration check: 0:03:16 ago on Fri 11 Nov 2022 09:21:36 PM CST.
Error: 
 Problem: conflicting requests
  - nothing provides perl(Net::IP) needed by freeradius-utils-3.0.25-1.oe2203.riscv64
(try to add '\''--skip-broken'\'' to skip uninstallable packages or '\''--nobest'\'' to use not only best candidate packages)'
+ '[' -z '' ']'
+ tmpfile='Last metadata expiration check: 0:03:16 ago on Fri 11 Nov 2022 09:21:36 PM CST.
Error: 
 Problem: conflicting requests
  - nothing provides perl(Net::IP) needed by freeradius-utils-3.0.25-1.oe2203.riscv64
(try to add '\''--skip-broken'\'' to skip uninstallable packages or '\''--nobest'\'' to use not only best candidate packages)'
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Fri Nov 11 21:24:54 2022 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Fri Nov 11 21:24:55 2022 - INFO  - Start to run test.
+ radcrypt --des test123
oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh: line 32: radcrypt: command not found
+ CHECK_RESULT 127 0 0 'radcrypt --des execution failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='radcrypt --des execution failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'radcrypt --des execution failed.'
+ LOG_ERROR 'radcrypt --des execution failed.'
+ message='radcrypt --des execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radcrypt --des execution failed.'
Fri Nov 11 21:24:56 2022 - ERROR - radcrypt --des execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 33'
+ message='oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 33'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 33'
Fri Nov 11 21:24:57 2022 - ERROR - oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 33
+ return 0
+ radcrypt --md5 test123
oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh: line 34: radcrypt: command not found
+ CHECK_RESULT 127 0 0 'radcrypt --md5 execution failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='radcrypt --md5 execution failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'radcrypt --md5 execution failed.'
+ LOG_ERROR 'radcrypt --md5 execution failed.'
+ message='radcrypt --md5 execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radcrypt --md5 execution failed.'
Fri Nov 11 21:24:59 2022 - ERROR - radcrypt --md5 execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 35'
+ message='oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 35'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 35'
Fri Nov 11 21:25:00 2022 - ERROR - oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 35
+ return 0
+ grep 'Password OK'
++ radcrypt --md5 test123
oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh: line 36: radcrypt: command not found
+ radcrypt --check test123
oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh: line 36: radcrypt: command not found
+ CHECK_RESULT 1 0 0 'radcrypt --check execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radcrypt --check execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radcrypt --check execution failed.'
+ LOG_ERROR 'radcrypt --check execution failed.'
+ message='radcrypt --check execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radcrypt --check execution failed.'
Fri Nov 11 21:25:01 2022 - ERROR - radcrypt --check execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 37'
+ message='oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 37'
Fri Nov 11 21:25:02 2022 - ERROR - oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 37
+ return 0
+ systemctl start radiusd
Failed to start radiusd.service: Unit radiusd.service not found.
+ SLEEP_WAIT 1
+ wait_time=1
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 1 --cmd '' --mode 1
+ echo 'Message-Authenticator = 0x00'
+ radeapclient -4 127.0.0.1 status testing123
+ grep 'Main loop: done'
oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh: line 40: radeapclient: command not found
+ CHECK_RESULT 1 0 0 'radeapclient -4 execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radeapclient -4 execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radeapclient -4 execution failed.'
+ LOG_ERROR 'radeapclient -4 execution failed.'
+ message='radeapclient -4 execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radeapclient -4 execution failed.'
Fri Nov 11 21:25:05 2022 - ERROR - radeapclient -4 execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 41'
+ message='oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 41'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 41'
Fri Nov 11 21:25:06 2022 - ERROR - oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 41
+ return 0
+ echo 'Message-Authenticator = 0x00'
+ grep 'Main loop: done'
+ radeapclient -6 '[::1]' status testing123
oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh: line 42: radeapclient: command not found
+ CHECK_RESULT 1 0 0 'radeapclient -6 execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radeapclient -6 execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radeapclient -6 execution failed.'
+ LOG_ERROR 'radeapclient -6 execution failed.'
+ message='radeapclient -6 execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radeapclient -6 execution failed.'
Fri Nov 11 21:25:07 2022 - ERROR - radeapclient -6 execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 43'
+ message='oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 43'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 43'
Fri Nov 11 21:25:08 2022 - ERROR - oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 43
+ return 0
+ echo 'Message-Authenticator = 0x00'
+ radeapclient -d /etc/raddb 127.0.0.1 status testing123
+ grep 'Main loop: done'
oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh: line 44: radeapclient: command not found
+ CHECK_RESULT 1 0 0 'radeapclient -d execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radeapclient -d execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radeapclient -d execution failed.'
+ LOG_ERROR 'radeapclient -d execution failed.'
+ message='radeapclient -d execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radeapclient -d execution failed.'
Fri Nov 11 21:25:10 2022 - ERROR - radeapclient -d execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 45'
+ message='oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 45'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 45'
Fri Nov 11 21:25:11 2022 - ERROR - oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 45
+ return 0
+ echo 'Message-Authenticator = 0x00'
+ radeapclient -D /usr/share/freeradius 127.0.0.1 status testing123
+ grep 'Main loop: done'
oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh: line 46: radeapclient: command not found
+ CHECK_RESULT 1 0 0 'radeapclient -D execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radeapclient -D execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radeapclient -D execution failed.'
+ LOG_ERROR 'radeapclient -D execution failed.'
+ message='radeapclient -D execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radeapclient -D execution failed.'
Fri Nov 11 21:25:12 2022 - ERROR - radeapclient -D execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 47'
+ message='oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 47'
Fri Nov 11 21:25:13 2022 - ERROR - oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 47
+ return 0
+ echo 'Message-Authenticator = 0x00'
+ radeapclient -f /tmp/test 127.0.0.1 status testing123
oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh: line 49: radeapclient: command not found
+ grep 'Main loop: done'
+ CHECK_RESULT 1 0 0 'radeapclient -f execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radeapclient -f execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radeapclient -f execution failed.'
+ LOG_ERROR 'radeapclient -f execution failed.'
+ message='radeapclient -f execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radeapclient -f execution failed.'
Fri Nov 11 21:25:14 2022 - ERROR - radeapclient -f execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 50'
+ message='oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 50'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 50'
Fri Nov 11 21:25:15 2022 - ERROR - oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 50
+ return 0
+ radeapclient -h
+ grep -i Usage
oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh: line 51: radeapclient: command not found
+ CHECK_RESULT 1 0 0 'radeapclient -h execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radeapclient -h execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radeapclient -h execution failed.'
+ LOG_ERROR 'radeapclient -h execution failed.'
+ message='radeapclient -h execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radeapclient -h execution failed.'
Fri Nov 11 21:25:16 2022 - ERROR - radeapclient -h execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 52'
+ message='oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 52'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 52'
Fri Nov 11 21:25:17 2022 - ERROR - oe_test_freeradius_freeradius-utils_radcryptAndRadeapclient.sh line 52
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Fri Nov 11 21:25:18 2022 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 9
+ test 9 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Nov 11 21:25:19 2022 - ERROR - failed to execute the case.
+ exit 9
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Fri Nov 11 21:25:20 2022 - INFO  - Start to restore the test environment.
+ systemctl stop radiusd
Failed to stop radiusd.service: Unit radiusd.service not loaded.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z Last metadata expiration check: 0:03:16 ago on Fri 11 Nov 2022 09:21:36 PM CST.
Error: 
 Problem: conflicting requests
  - nothing provides perl(Net::IP) needed by freeradius-utils-3.0.25-1.oe2203.riscv64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages) ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile 'Last metadata expiration check: 0:03:16 ago on Fri 11 Nov 2022 09:21:36 PM CST.
Error: 
 Problem: conflicting requests
  - nothing provides perl(Net::IP) needed by freeradius-utils-3.0.25-1.oe2203.riscv64
(try to add '\''--skip-broken'\'' to skip uninstallable packages or '\''--nobest'\'' to use not only best candidate packages)'
Traceback (most recent call last):
  File "/root/mugen-riscv/libs/locallibs/rpm_manage.py", line 173, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/mugen-riscv/libs/locallibs/rpm_manage.py", line 144, in rpm_remove
    with open(tmpfile, "r") as f:
OSError: [Errno 36] File name too long: "Last metadata expiration check: 0:03:16 ago on Fri 11 Nov 2022 09:21:36 PM CST.\nError: \n Problem: conflicting requests\n  - nothing provides perl(Net::IP) needed by freeradius-utils-3.0.25-1.oe2203.riscv64\n(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)"
+ '[' 0 -ne 0 ']'
+ rm -rf /etc/raddb
+ rm -rf /var/log/radius
+ rm -rf /tmp/test
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Fri Nov 11 21:25:26 2022 - INFO  - End to restore the test environment.
